index.js

```diff
@@ -4,12 +4,52 @@ import assert from 'node:assert/strict'
+
+    console.log("pGpub")
+    console.log(pGpub)
+    console.log("pGmessage")
+    console.log(pGMessage)
+    console.log("pGsign")
+    console.log(pGsign)
+    console.log("pGpriv")
+    console.log(pGpriv)
+
+    const pGpubUint8Array = Uint8Array.from(atob(pGpub), char => char.charCodeAt(0));
+    const pGprivUint8Array = Uint8Array.from(atob(pGpriv), char => char.charCodeAt(0));
+    const pGsignUint8Array = Uint8Array.from(atob(pGsign), char => char.charCodeAt(0));
```

Comment:
WHat are you trying to do here? pG{pub,priv,sign} are base64-encoded, not ascii. Then you should use `Buffer.from(<base64-encoded-string>).toString('base64')` to get the corresponding UInt8Array.

Reply:
those are initially base64 because were copied from the Playground to make possible to check if a signature validated in the Playground, validates also on nodejs, but while using the same base64, it fails on nodejs while pass on Playground.

the previous portion that has the base64 as output is the opposite, keys and signature are created by nodejs and verify locally, but when the base64 output is copy and pasted to Playground it fails the verification